src/EditorFeatures/Core/Remote/RemoteHostOptions.cs

```diff
@@ -9,11 +9,11 @@ namespace Microsoft.CodeAnalysis.Remote
     internal sealed class RemoteHostOptions
     {
         // use 64bit OOP
-        public static readonly Option2<bool> OOP64Bit = new("InternalFeatureOnOffOptions_OOP64Bit", defaultValue: true);
+        public static readonly Option2<bool> OOP64Bit = new("dotnet_enable_oop_64_bit", defaultValue: true);
```

Comment:
>dotnet_enable_oop_64_bit [](http://example.com/codeflow?start=61&length=24)

This option now means "run code analysis in separate process", nothing to do with 64-bit anymore. We should rename the option field as well to match that.

`dotnet_code_analysis_in_separate_process` would seem like a reasonable name.